### PR TITLE
Custom channel representation: add SAMSHA channel

### PR DESCRIPTION
**Pull request by erikos**
Only add a thumbnail and an icon for the SAMSHA channel.

https://github.com/endlessm/kolibri-explore-plugin/issues/742

### CODE REVIEW

File: kolibri_explore_plugin/assets/src/customApps.js [hunk 1]
```diff
@@ -218,6 +218,7 @@ export const ChannelCardThumbnails = {
   '057f871caa405ec29d62ba0523c193d7': 'music.jpg',
   '914fee213ee146de869016c287116b23': 'novels.jpg',
   '59bb2e5a3d2e5e3b85b87d9ab4daa2f3': 'reading.jpg',
+  a8e6591f1afa426d859318a0a29d1237: 'samhsa.png',
```
Comment (GeorgesStavracas) [inline kolibri_explore_plugin/assets/src/customApps.js:221]: I think the hash needs to be quoted, `'a8e6591f1afa426d859318a0a29d1237'`

File: kolibri_explore_plugin/assets/src/customApps.js [hunk 2]
```diff
@@ -281,6 +282,7 @@ export const ChannelCardIcons = {
   '057f871caa405ec29d62ba0523c193d7': 'music.png',
   '914fee213ee146de869016c287116b23': 'novels.png',
   '59bb2e5a3d2e5e3b85b87d9ab4daa2f3': 'reading.png',
+  a8e6591f1afa426d859318a0a29d1237: 'samhsa.png',
```
Comment (GeorgesStavracas) [inline kolibri_explore_plugin/assets/src/customApps.js:285]: I think the hash needs to be quoted, `'a8e6591f1afa426d859318a0a29d1237'`, here too

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]

**Comment by jprvita** [comment]
> I think the hash needs to be quoted

Would it be possible to check that in the lint phase of CI?

**Comment by erikos** [comment]
> > Would it be possible to check that in the lint phase of CI?
> 
> Might be worthy, yes

Check the other occasions in the same file - ids starting with a character do not have to be quoted. I am sure Manuel has a nice story to go with, but I do not remember the reason.

**Pull request merged**